<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>29</y>
      <width>551</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>8</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>171</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Welcome</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>221</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Log In</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>113</width>
         <height>29</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>113</width>
         <height>29</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>221</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Sign Up</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>113</width>
         <height>29</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>113</width>
         <height>29</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Main Menu</string>
      </property>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>131</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Public Calendar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>131</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Private Calendar </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>131</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Timetable</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>131</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Shared Calendar</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>231</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Create Timetable</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Table Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Public</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Private</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Shared</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>201</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Tables</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>151</width>
         <height>29</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Select Table</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>0</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>331</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Create Event</string>
      </property>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>194</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>91</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Time</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>End Time</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>194</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>113</width>
         <height>29</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>290</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Event Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>End</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Details</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>start</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>end</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>details</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
